### PR TITLE
Add default views to blogging reminders

### LINKED ISSUE #14715: Add basic UI setup for the first blogging reminders screen
In the first blogging reminders screen we want to show: 
- centered illustration
- title
- text
- primary button

The copys and the resources will yet change so the architecture needs to be easily extensible.

### PR DESCRIPTION
**Pull request by planarvoid**
Fixes #14715

This PR adds default views for the first screen of the blogging reminder prompt. I've used the updates resources and copys.

To test:
- Clear app data and restart
- Log in
- Turn on the blogging reminders flag in Test feature configuration
- Create a new post
- Notice the bottom sheet contains the expected views

Screenshot 2021-06-01 at 11 48 04

## Regression Notes
1. Potential unintended areas of impact
None

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersAdapter.kt
Comment (ravishanker) [inline WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersAdapter.kt:23]: Just curious!  Why is this modeled as an Adapter with ViewHoders?  Does the content text come from backend here!
Is it is like airbnb [epoxy](https://github.com/airbnb/epoxy)
There's a similar [ConcatAdapter](https://medium.com/androiddevelopers/merge-adapters-sequentially-with-mergeadapter-294d2942127a) in the works but still in Alpha I guess.

Comment (planarvoid) [reply WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersAdapter.kt:23]: I tend to model all the UI as a list. It makes it much easier to unit test the UI state, makes building a UI much more flexible and removes a lot of the logic from the Views. I've never used epoxy. Also from a quick look at ConcatAdapter I don't see any benefit over multiple view holders. Are there any?

**Review by renanferrari — APPROVED** [review]
Thanks for the changes! Code is looking good and I can confirm the app is working as expect.

LGTM 👍

Are you also going to ask for a design review? If not, feel free to merge this once we merge the base branch.

File: WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersAdapter.kt
Comment (ravishanker) [reply WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersAdapter.kt:23]: Not really about advantage.  I found your approach similar to epoxy.  I've used epoxy in cases where content comes from backend and screens are built dynamically.  Wondering, if that was the reason why you chose to go with this approach.

ConcatAdapter is similar to airbnb epoxy.  ConcatAdapter allows to combine multiple types of adapters with their own data sources and also reuse viewholders across adapters too. 

Of course, all that might become obsolete when we start using Jetpack Compose components!

**Review by osullivanchris — APPROVED** [review]
Hey it looks good @planarvoid 

I checked it in light mode, dark mode, portrait, landscape. Having trouble with Android Studio right now and wasn't able to check it out on tablet.

There's a couple of minor differences from the mock. But noting reasons why I think the implementation is more correct anyway:
- the button height is less in the build - I'm assuming you used default height as other buttons in which case 👍 
- the sheet does not have rounded corners - I think we are always doing this when sheet is not expandable in which case 👍 

One other thing I tested was publishing a story post. In that case, the reminder did not occur. Is that expected?

**Comment by planarvoid** [comment]
> One other thing I tested was publishing a story post. In that case, the reminder did not occur. Is that expected?

good question, let's discuss this in the chat 👍

**Pull request merged**
